Formula/yelp-tools.rb

```diff
@@ -15,30 +16,47 @@ class YelpTools < Formula
   end
 
   depends_on "gettext" => :build
-  depends_on "intltool" => :build
   depends_on "itstool" => :build
-  depends_on "libxml2" => :build
-  depends_on "libxslt" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "gtk+3"
+  depends_on "libxml2"
+  depends_on "python@3.9"
+
+  uses_from_macos "libxslt"
```

Comment:
Wasn't sure if better to keep Homebrew's version of `libxslt` or use macOS.
Switched based on common choice in formulae
(55 uses from macOS, while 4 [including `yelp-tools`] depends on Homebrew version).

I did leave `libxml2` though...